test/integration/domain.spec.ts

```diff
@@ -264,34 +264,45 @@ describe('Domain Layer', () => {
         },
         totalSupply: totalRep,
       },
-      schemes: [
-        {
+      gpQueues: [],
+    });
+
+    expect(dao.schemes).toContainEqual(
+      {
           address: addresses.UpgradeScheme.toLowerCase(),
           dao: {
             id: addresses.Avatar.toLowerCase(),
           },
         },
-        {
+    );
+
+    expect(dao.schemes).toContainEqual(
+      {
           address: addresses.GlobalConstraintRegistrar.toLowerCase(),
           dao: {
             id: addresses.Avatar.toLowerCase(),
           },
         },
-        {
+    );
+
+    expect(dao.schemes).toContainEqual(
+      {
           address: addresses.ContributionReward.toLowerCase(),
           dao: {
             id: addresses.Avatar.toLowerCase(),
           },
         },
-        {
+    );
```

Comment:
Why so many new expect statements?

Reply:
to ignore the order of the array entries.